README.md

```diff
@@ -421,4 +421,4 @@ A way to mount your components in the same style as Roact.
 
 #### Links
 - [**Repository**](https://github.com/howmanysmall/ReactTree)
-- [**Wally**](https://wally.run/package/howmanysmall/react-tree)
\ No newline at end of file
+- [**Wally**](https://wally.run/package/howmanysmall/react-tree)
```

Comment:
This is just the GitHub web editor adding an extra \n at the end of the file. Can remove if you want.